Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -349,13 +349,19 @@ void DirectoryView::add_path_to_history(const StringView& path)
 
 void DirectoryView::open(const StringView& path)
 {
-    if (model().root_path() == path) {
+    String real_path;
+
+    char* buffer = realpath(path.characters_without_null_termination(), nullptr);
+    real_path = buffer;
+    free(buffer);
```

Comment:
This is essentially a re-implementation of `Core::File::real_path_for()`, so let's use that :)

```suggestion
    auto real_path = Core::File::real_path_for(path);
```